pkg/segment/writer/segstore.go

```diff
@@ -54,6 +54,9 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const MaxAgileTreeNodeCountForAlloc = 8_066_000 // for atree to do allocations
+// For Last wip we do not know how many nodes this wip will add, hence we
+// leave a room for one wip's worth of recs.
 const MaxAgileTreeNodeCount = 8_000_000
```

Comment:
nit: are these comments all for the new constant? If so, can you put them above it instead of below it?

Reply:
fixed